common/src/main/java/net/dodogang/plume/Plume.java

```diff
@@ -43,30 +45,30 @@ public static void initialize() {
         if (runDevTests) {
             LOGGER.log(Level.INFO, "Development environment detected! Running dev initialize.");
 
-            ItemGroup itemGroup = new TabbedItemGroup("plume_test",
-                (id) -> {
-                    String ns = id.getNamespace();
-                    return ImmutableList.of(
-                        TabbedItemGroup.createTab(Blocks.DIRT, ns, "unga"),
-                        TabbedItemGroup.createTab(Blocks.BLACK_CONCRETE, ns, "bunga")
-                    );
-                }, () -> new ItemStack(Blocks.STONE)
-            );
+            ItemGroup itemGroup = new TabbedItemGroup("plume_test", (id) -> {
+                String ns = id.getNamespace();
+                return ImmutableList.of(
+                    TabbedItemGroup.createTab(Blocks.DIRT, ns, "unga"),
+                    TabbedItemGroup.createTab(Blocks.BLACK_CONCRETE, ns, "bunga"));
+            }, () -> new ItemStack(Blocks.STONE));
 
-            BlockRegistryBatch blocks = new BlockRegistryBatch(MOD_ID).setDefaultItemSettings(new Item.Settings().group(itemGroup));
+            BlockRegistryBatch blocks =
+                new BlockRegistryBatch(MOD_ID).setDefaultItemSettings(new Item.Settings().group(itemGroup));
 
             TEST_BLOCK = blocks.add("test_block", new Block(AbstractBlock.Settings.copy(Blocks.STONE)));
-            TEST_BEAM_BLOCK = blocks.add("test_beam_block", new BeamBlock(AbstractBlock.Settings.copy(Blocks.OAK_PLANKS)));
-            TEST_CEILING_PLANT_BLOCK = blocks.add("test_ceiling_plant_block", new CeilingPlantBlock(AbstractBlock.Settings.copy(Blocks.GRASS)));
-            TEST_TALL_CEILING_PLANT_BLOCK = blocks.add("test_tall_ceiling_plant_block", new TallCeilingPlantBlock(AbstractBlock.Settings.copy(Blocks.TALL_GRASS)));
-            TEST_TALLER_PLANT_BLOCK = blocks.add("test_taller_plant_block", new TallerPlantBlock(AbstractBlock.Settings.copy(Blocks.TALL_GRASS)));
+            TEST_BEAM_BLOCK =
+                blocks.add("test_beam_block", new BeamBlock(AbstractBlock.Settings.copy(Blocks.OAK_PLANKS)));
+            TEST_CEILING_PLANT_BLOCK = blocks.add(
+                "test_ceiling_plant_block", new CeilingPlantBlock(AbstractBlock.Settings.copy(Blocks.GRASS)));
+            TEST_TALL_CEILING_PLANT_BLOCK = blocks.add(
+                "test_tall_ceiling_plant_block",
+                new TallCeilingPlantBlock(AbstractBlock.Settings.copy(Blocks.TALL_GRASS)));
```

Comment:
I wonder if I could make the trailing `);` go onto the next line

Reply:
I hate the newline after the equals aaaaaaaa
also, same with the builder above, this is too short in my opinion to attempt to format nicely, you may as well leave these as single lines